File: CS50/pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>



int main(void)
{
    int n;
    bool bool1;
    bool bool2;
    bool validity;
    do
    {
        n = get_int("Number of rows: ");
        bool1 = n <= 0;
        bool2 = n > 23;
        validity = bool1||bool2;
    }while (validity);
    for (int i = 0; i < n; i++)
        {
            int spaces = n-i-1;
            int hashes = n-spaces;

            for (int j = 0; j <= spaces; j++)
            {
                printf(" ");
            }
            for (int k = 0; k <= hashes; k++)
            {
                printf("#");
            }
            printf("\n");
        }

}

File: CS50/pset6/mario/mario.py
import cs50

while True:
    height = cs50.get_int("Number of rows: ")
    bool1 = height <= 0
    bool2 = height > 32
    if bool1 or bool2:
        break
    else:
        for i in range(height):
            spaces = height - i - 1
            hashes = height - spaces
            print(" " * spaces, end = "")
            print("#" * hashes)


